### ISSUE TITLE
Navigation Bar rendering

### ISSUE THREAD

**Issue by cmcfadden** [UMN-LATIS/elevator-ui#48]
Endpoint which should vend the information necessary to render the navbar:
`https://dev.elevator.umn.edu/defaultinstance/home/getInstanceNav`

That'll return JSON in the structure below - annotations added.

The "recent collections" and "recent drawers" are a little clunky, and I'm not sure we need to keep them. They currently only update when a user explicitly goes through the "All collections" or "All drawers" interface, so if they effectively browse collections via a search or something, it won't be reflected. 

```
{
    "pages": [ -- pages for building the navbar. Children pages will follow the same format, and are fed into the pulldowns in the current site. Order of the items in the array corresponds to order drawn in the nav
        {
            "title": "Another Page",
            "id": 17,
            "children": []
        },
        {
            "title": "test",
            "id": 14,
            "children": []
        }
    ],
    "userIsloggedIn": true, --  info below describes the user if logged in. These keys will always be populated, they just might be set to false
    "userCanCreateDrawers": true,
    "userCanManageAssets": true,
    "userId": 1,
    "userDisplayName": "Admin",
    "userIsAdmin": true,
    "userIsSuperAdmin": true,
    "instanceName": "defaultinstance", not sure if we need this or if we'll have it via the config object
    "instanceId": 1, -- not sure if we need this or if we'll have it via the config object
	"instanceHasLogo": false,
    "recentDrawers": [ -- array of drawers most recently accessed by the user
        {
            "id": 440,
            "title": "James drawer"
        }
		-- snip --
    ],
    "recentCollections": [ -- array of collections most recently accessed by the user via the collection browse list
        {
            "id": 41,
            "title": "Speeches 2 - Don't use"
        }
        -- snip --
    ],
    "contact": "mailto:mcfa0086@umn.edu", -- This can be either a mailto: or a URL (http://). Feeds the "contact us" link under help.
    "useCentralAuth": true, -- determines whether or not we show the "University Login" link in addition to the local user login
	"centralAuthLabel": "University", -- determines the label of the central auth link. Could be "St. Olaf Login" or "College Login" or whatever"
	"collections": [  -- nested hierarchy of all collections that the user has permissions to see
        {
            "id": 74,
            "title": "API Import",
            "children": [
                {
                    "id": 75,
                    "title": "Bennington Test Collection"
                }
            ]
        }
		-- snip --
	]

}
```

**Comment by cmcfadden** [comment]
Updated with
`  "instanceHasLogo": false,`

If it's true, the instance logo can be found at:
`/assets/instanceAssets/<instanceId>.png`

**Comment by jxjj** [comment]
Thanks – Will the logo always be a png?

**Comment by cmcfadden** [comment]
Yep

**Comment by jxjj** [comment]
Resolved in #50
